### ISSUE TITLE
Blockmarket desktop not syncing?

### ISSUE THREAD

**Issue by stepwi4** [syscoin/blockmarket-desktop-public#293]
Downloaded Blockmarket desktop for windows but just shows "No blocks found" under Recent Blocks. I'm thinking it's not syncing. Any ideas?

**Comment by johnp1954** [comment]
Please go to the help channel and ask

**Comment by stepwi4** [comment]
What help channel?

**Comment by stepwi4** [comment]
I'm new to all this so I'm trying to figure it all out.

**Comment by johnp1954** [comment]
In Slack you are a member look at the channels on the left

**Comment by stepwi4** [comment]
I've heard of Slack but not sure what it is. You say I am a member?

**Comment by johnp1954** [comment]
Can you give me your Email and I will invite you then we can sort this

**Comment by stepwi4** [comment]
stepwi4@gmail.com

**Comment by johnp1954** [comment]
Sent just ask for johnp when you join

**Comment by solidcloudio** [comment]
Can you add me to slack also
zapbbx@gmail.com

**Comment by Sporadic92** [comment]
Add me to slack as weel Jakelawlor92@gmail.com

**Comment by cobbju** [comment]
Same issue, can you add me to slack as well kratosebay@gmail.com

**Comment by johnp1954** [comment]
Invites sent

**Comment by flynnp3** [comment]
can you email me too, flynnp9@tcd.ie , sent 90 coins to wallet last night and no sync bar

**Comment by dwasyluk** [comment]
Duplicate of #292 - caused by our sys seeds being down. Has since been fixed, please comment if still having issues.

**Comment by kzrt66** [comment]
Hi all.
I am new to this space, and not tech savvy. Please help.

I downloaded Blockmarket and synced yesterday. When I opened it, it had to be synced all over again.
Is it a normal procedure, or did I make a mistake? (As a matter of fact, when I tried to open, a error message window popped up.)

Thank you very much for your help.

**Comment by kzrt66** [comment]
Hi all. 
I am new to this space and not tech savvy at all.  Your help is most appreciated.

I installed Blockmarket yesterday, and syncing completed after a few hours.
When I opened it today, I only found 0% sync, and had to wait for the syncing to complete again. It appears syncing is much faster than yesterday.

Q1. Do I have to sync from the beginning every time I open Blockmarket?
Q2. There was an error message window when I opened Blockmarket although I was able to go to Home/Dashboard. Do I need to fix them all? And how?

Thank you very much!
